libraryversions.toml

```diff
@@ -292,7 +292,7 @@ WEAR_TILES = { group = "androidx.wear.tiles", atomicGroupVersion = "versions.WEA
 WEAR_WATCHFACE = { group = "androidx.wear.watchface", atomicGroupVersion = "versions.WEAR_WATCHFACE" }
 WEBKIT = { group = "androidx.webkit", atomicGroupVersion = "versions.WEBKIT" }
 WINDOW = { group = "androidx.window", atomicGroupVersion = "versions.WINDOW" }
-WINDOW_CORE = { group = "org.jetbrains.window.core", atomicGroupVersion = "versions.WINDOW", overrideInclude = [ ":window:window-core" ]  }
+WINDOW_CORE = { group = "org.jetbrains.androidx.window.core", atomicGroupVersion = "versions.WINDOW", overrideInclude = [ ":window:window-core" ]  }
```

Comment:
```suggestion
WINDOW_CORE = { group = "org.jetbrains.androidx.window", atomicGroupVersion = "versions.WINDOW", overrideInclude = [ ":window:window-core" ]  }
```

Reply:
core is the name of the module, not of the group

Reply:
Also, to be consistent with other libraries, we should move it after `SAVEDSTATE_JETBRAINS` and rename to `WINDOW_CORE_JETBRAINS`

Reply:
agree, but then why are the next lines:
```
WINDOW_EXTENSIONS = { group = "androidx.window.extensions", atomicGroupVersion = "versions.WINDOW_EXTENSIONS" }
WINDOW_EXTENSIONS_CORE = { group = "androidx.window.extensions.core", atomicGroupVersion = "versions.WINDOW_EXTENSIONS_CORE" }
```